go test failure "panic: send on closed channel"

https://dev.azure.com/ucfconsort/ucx/_build/results?buildId=32853&view=logs&j=3326af28-725b-5a76-d9b2-a6afcb2c442d&t=7704eb91-d2f0-594b-7192-7aca9f7d6696

https://dev.azure.com/ucfconsort/0b36e3f0-8ab9-4a48-b68b-4b2350e02c88/_apis/build/builds/32853/logs/225

```
=== RUN   TestUcpContext
--- PASS: TestUcpContext (0.03s)
=== RUN   TestUcpEpTag
    endpoint_test.go:109: Testing tag host -> host
    endpoint_test.go:109: Testing tag host -> cuda
    endpoint_test.go:109: Testing tag cuda -> host
    endpoint_test.go:109: Testing tag cuda -> cuda
--- PASS: TestUcpEpTag (2.59s)
=== RUN   TestUcpEpAm
    endpoint_test.go:176: Testing AM host -> host
panic: send on closed channel

goroutine 18 [running]:
_/scrap/azure/agent-01/AZP_WORKSPACE/1/s/bindings/go/tests.TestUcpEpAm.func2(0x7fd86fff9186, 0xb, 0xc00005c400, 0xc00021d728, 0x20300000000000)
	/scrap/azure/agent-01/AZP_WORKSPACE/1/s/bindings/go/tests/endpoint_test.go:225 +0x1b4
ucx.ucxgo_amRecvCallback(0xb, 0x7fd86fff9186, 0xb, 0x7fd86fff9142, 0xb, 0x7fd8a7674b20, 0x7fd8d102b218)
	/scrap/azure/agent-01/AZP_WORKSPACE/1/s/bindings/go/src/ucx/callbacks.go:101 +0x13c
ucx._Cfunc_ucp_worker_progress(0x23f3000, 0xc000000000)
	_cgo_gotypes.go:1070 +0x45
ucx.(*UcpWorker).Progress.func1(0xc000100078, 0x0)
	/scrap/azure/agent-01/AZP_WORKSPACE/1/s/bindings/go/src/ucx/worker.go:124 +0x59
ucx.(*UcpWorker).Progress(0xc000100078, 0xc00011e060)
	/scrap/azure/agent-01/AZP_WORKSPACE/1/s/bindings/go/src/ucx/worker.go:124 +0x2b
_/scrap/azure/agent-01/AZP_WORKSPACE/1/s/bindings/go/tests.progressThread(0xc00011e060, 0xc000100078)
	/scrap/azure/agent-01/AZP_WORKSPACE/1/s/bindings/go/tests/utils.go:87 +0x2d
created by _/scrap/azure/agent-01/AZP_WORKSPACE/1/s/bindings/go/tests.TestUcpEpAm
	/scrap/azure/agent-01/AZP_WORKSPACE/1/s/bindings/go/tests/endpoint_test.go:232 +0x8dc
exit status 2
FAIL	_/scrap/azure/agent-01/AZP_WORKSPACE/1/s/bindings/go/tests	3.303s
make: *** [test] Error 1
make: Leaving directory `/scrap/azure/agent-01/AZP_WORKSPACE/1/s/build/bindings/go'
##[error]Bash exited with code '2'.
```

--- a/bindings/go/tests/endpoint_test.go
+++ b/bindings/go/tests/endpoint_test.go
@@ -261,6 +261,8 @@ func TestUcpEpAm(t *testing.T) {
 			}
 		}
 
+		for len(requests) != cap(requests) {
+		}
 		close(requests)
 		for req := range requests {
 			for req.GetStatus() == UCS_INPROGRESS {

